Makefile

```diff
@@ -299,14 +299,17 @@ debug-jupyter-notebook: build-release-image-jupyter ## Debug jupyter notebook
 	docker run \
 		-it \
 		--rm \
+		--user=root \
 		--publish="$(jupyter_notebook_port):8888" \
 		--volume="$(CURDIR)/bindings/python/docs:/home/jovyan/docs" \
 		--volume="$(CURDIR)/tutorials/python/notebooks:/home/jovyan/tutorials" \
```

Comment:
This also gets chown'd, no? Need to clean this up as well?

Reply:
I'm not sure what you mean by this? This only executes the docker run command as root so that I'm allowed to chown the home/jovyan folder in the first place right? I'm not sure it does anything else that is "lasting" and needs to be cleaned up? Lmk if I'm wrong

Reply:
Oops I totally commented the wrong line 🤦 

I meant this second volume:
"$(CURDIR)/tutorials/python/notebooks:/home/jovyan/tutorials"

Since we `chown` the entire jovyan dir, this dir will be `chown`ed as well.

Reply:
Oh I see what you're saying. Yes that's correct, and I addressed it with another cleanup command after the docker image is stopped.